new-readme.md

```diff
@@ -81,133 +83,141 @@ npm run dev
 
 ### <a name="examples">Examples</a>
 
-Example setup is similar with new application creation, but it has `--example` or `-e` flag with example name. You can start working with reSolve example just in **2 minutes**.
+You can get a sample app using the `create-resolve-app` tool with the `-e` flag followed by an example name. Use the `create-resolve-app -h` command to list the available examples.
 
 <p align="center"><img src="https://github.com/reimagined/resolve/blob/feature/new_readme/readme-example-installation.gif"></p>
 
-The created example is hosted at http://localhost:3000/ and http://<your_ip>:3000/ (you can [change your url settings](https://github.com/reimagined/resolve/blob/master/docs/API%20References.md)). Also you can look at [**tutorials**](#tutorials) for some examples.
+The following tools are used to get a sample **reSolve** aplication:
+
+* [npx](https://www.npmjs.com/package/npx)
+
+    ```sh
+    npx create-resolve-app todo-example -e todo
+    ```
+
+* [yarn](https://yarnpkg.com/lang/en/)
+
+    ```sh
+    yarn create resolve-app todo-example -e todo
+    ```
 
-#### Using npx
-```bash
-npx create-resolve-app todo-example -e todo
-```
+* [npm](https://www.npmjs.com/)
 
-#### Using yarn
-```bash
-yarn create resolve-app todo-example -e todo
-```
+    ```sh
+    npm i -g create-resolve-app
+    create-resolve-app todo-example -e todo
+    ```
 
-#### Using npm
-```bash
-create-resolve-app todo-example -e todo
-```
+The created application is accessible using the http://localhost:3000/ and `http://<your_ip>:3000` URLs (you can [change your URL settings](https://github.com/reimagined/resolve/blob/master/docs/API%20References.md)).
+
+The `create-resolve-app` can is capable to create the following apps:
 
 * [**hello-world**](https://github.com/reimagined/resolve/tree/master/examples/hello-world)
 
-It's a simple empty example that can be used like **reSolve application boilerplate**.
+    An empty app that can be used as a template for any **reSolve** application (created by default).
 
 <p align="center"><img src="https://github.com/reimagined/resolve/blob/feature/new_readme/readme-hello-world-example.png"></p>
 
 * [**top-list**](https://github.com/reimagined/resolve/tree/master/examples/top-list)
 
-This example demonstrates **reactive read-models** that leads to client updating without page reloads. You can learn more about [read-models in documentation](https://github.com/reimagined/resolve/blob/master/docs/Read%20Model.md)
+    This example demonstrates how to work with the [reactive read-models](https://github.com/reimagined/resolve/blob/master/docs/Read%20Model.md) used to update information on the web page without reloading it.
```

Comment:
No
Both view-models and read-models (and React and Redux) are used to update information on the web page without reloading it.
This example is about reactive read-models, that are update state automatically send diff from server without any actions (i mean client-server requests)

Reply:
@TatyanaRyzh  ?

Reply:
This example demonstrates how to update UI data on external events unrelated to user actions.